### ISSUE TITLE
`is_isomorphic` doesn't work with PyGraph objects

### ISSUE THREAD

**Issue by quantumjim** [Qiskit/retworkx#348]
### Information

- **retworkx version**: 0.8.0
- **Python version**: 3.9.1
- **Rust version**: 1.50.0
- **Operating system**: OSX

### What is the current behavior?

Running the code below results in the error

```
TypeError: Can't convert <retworkx.PyGraph object at 0x7feda8dab6c0> to PyDiGraph
```

### What is the expected behavior?

It should work ;)

### Steps to reproduce the problem

```
import retworkx

g_a = retworkx.PyGraph()
g_b = retworkx.PyGraph()

g_a.add_node(0)
g_a.add_node(1)
g_a.add_edge(0,1,':)')

g_b.add_node(0)
g_b.add_node(1)
g_b.add_edge(0,1,':(')

retworkx.is_isomorphic(g_a,g_b)
```

**Comment by mtreinish** [comment]
This limitation was actually already fixed in the 0.9.0 release which was released on Tuesday. (it's in the release notes: https://qiskit.org/documentation/retworkx/release_notes.html#relnotes-0-9-0 ) Can you give 0.9.0 a try and confirm it fixes it for you?

**Comment by quantumjim** [comment]
Yep, that it does. Thanks!
